open-banking-brasil-financial-api-1_ID1-ptbr.md

```diff
@@ -321,21 +320,21 @@ Para TLS, endpoints do Servidor de Autenticação e endpoints do Servidor de Rec
 
 ### Introdução  {#authmechintro}
 
-Os mecanismos existentes para gerenciar adequadamente o acesso aos recursos definidos em [RFC6749] são insuficientes para atender aos requisitos de um ecossistema de compartilhamento de dados moderno. Aproveitar strings de escopo estático não fornece aos consumidores controle de granularidade suficiente para compartilhar com terceiros. O Open Banking Brasil optou por implementar uma [API de consentimento] (https://openbanking-brasil.github.io/areadesenvolvedor/swagger/swagger_consents_apis.yaml) como um recurso protegido OAuth 2.0 que pode ser usado para gerenciar o acesso granular aos recursos. A referência ao recurso de consentimento será transmitida como parte de um escopo de recurso dinâmico OAuth 2.0.
+Os mecanismos existentes para gerenciar adequadamente o acesso aos recursos definidos em [RFC6749] são insuficientes para atender aos requisitos de um ecossistema de compartilhamento de dados moderno. Aproveitar strings de escopo estático não fornece aos consumidores controle de granularidade suficiente para compartilhar com terceiros. O Open Banking Brasil optou por implementar uma [API de consentimento](https://openbanking-brasil.github.io/areadesenvolvedor/swagger/swagger_consents_apis.yaml) como um recurso protegido OAuth 2.0 que pode ser usado para gerenciar o acesso granular aos recursos. A referência ao recurso de consentimento será transmitida como parte de um escopo de recurso dinâmico OAuth 2.0.
 
 ### Definição de Escopo de Consentimento Dinâmico  {#consent}
 
 Este perfil define o escopo dinâmico do OAuth 2.0 "consentimento" da seguinte maneira:
 
-* string 'consentimento'; e
+* string 'consent'; e
 * delimitador de dois pontos ":"; e
 * Consent API REST Resource Id retornado por uma criação bem-sucedida de [Open Banking Consent Resource](https://openbanking-brasil.github.io/areadesenvolvedor/#fase-2-apis-do-open-banking-brasil-api-consentimento);
 
 Adicionalmente:
 
 * Consent Resource Id deve incluir caracteres seguros para url;
-* Consent Resource Id deve ter namespace;
-* Consent Resource Id deve ter propriedades de um nonce [Nonce](https://pt.wikipedia.org/wiki/Nonce);
+* Consent Resource Id deve ser "namespaced";
+* Consent Resource Id deve ter propriedades de um _nonce_ [Nonce](https://pt.wikipedia.org/wiki/Nonce);
```

Comment:
Refer to the OIDC definition of nonce: https://openid.net/specs/openid-connect-core-1_0.html#NonceNotes if it is required however as this specification inherits everything from OIDC core it should be superfluous. e.g The requirements for nonce aren't defined in wikipedia.

Reply:
This wasn't caused by me :-) but I´ve change the reference as you suggest.

Reply:
Thanks

Reply:
Also please add yourself to the contributors section. I'll add you to the English version as well when this is done. So that the next editions of both specs get the same contribution recognition. Thanks again!